rsvim_core/src/ui/widget/command_line.rs

```diff
@@ -63,94 +64,23 @@ pub struct CommandLine {
   indicator_id: TreeNodeId,
   content_id: TreeNodeId,
   cursor_id: Option<TreeNodeId>,
+  message_id: TreeNodeId,
 
   text_contents: TextContentsWk,
 
   viewport: ViewportArc,
   cursor_viewport: CursorViewportArc,
 }
 
-impl CommandLine {
-  pub fn new(shape: IRect, text_contents: TextContentsWk) -> Self {
```

Comment:
Let's keep the "new" constructor, it should still be the logical ui widget for "command line".

Please take a look at window: https://github.com/rsvim/rsvim/blob/fd819f2dc1c843d341b28d35810155f32e38cd1e/rsvim_core/src/ui/widget/window.rs#L57